### PR TITLE
PMM-6944 New endpoint for get kutebernetes cluster.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1340

### CODE REVIEW

File: api/managementpb/dbaas/kubernetes.proto [hunk 1]
```diff
@@ -63,6 +63,24 @@ message UnregisterKubernetesClusterRequest {
...
+    }
+  ];
+}
+
+message GetKubernetesClusterResponse {
+  // Kubernetes auth.
+  KubeAuth kube_auth = 1 [
+    (validator.field) = {
+      msg_exists: true
+    }
```
Comment (askomorokhov) [inline api/managementpb/dbaas/kubernetes.proto:80]: I guess we do not need validator on response

**Review by askomorokhov — CHANGES_REQUESTED** [review]

File: api/managementpb/dbaas/kubernetes.proto (hunk 1 above)
Comment (JiriCtvrtka) [reply api/managementpb/dbaas/kubernetes.proto]: Fixed. Thanks.

**Review by askomorokhov — APPROVED** [review]
👍

**Comment by JiriCtvrtka** [comment]
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1324

File: api/managementpb/dbaas/kubernetes.proto [hunk 2]
```diff
@@ -86,4 +100,11 @@ service Kubernetes {
       body: "*"
     };
   }
+  // GetKubernetesCluster return KubAuth with Kubernetes config.
```
Comment (BupycHuk) [inline api/managementpb/dbaas/kubernetes.proto:103]:
```suggestion
  // GetKubernetesCluster return KubeAuth with Kubernetes config.
```

**Review by BupycHuk — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**